src/components/web3/connectors.ts

```diff
@@ -1,138 +1,102 @@
-  [CHAINS.OPTIMISM_KOVAN]: process.env.REACT_APP_RPC_URL_OPTIMISM_KOVAN || getChainById(CHAINS.OPTIMISM_KOVAN).metamask.rpcUrls[0],
-  [CHAINS.POLYGON_TESTNET]: process.env.REACT_APP_RPC_URL_POLYGON_MUMBAI || getChainById(CHAINS.POLYGON_TESTNET).metamask.rpcUrls[0],
-  [CHAINS.BSC_TESTNET]: process.env.REACT_APP_RPC_URL_BSC_TESTNET || getChainById(CHAINS.BSC_TESTNET).metamask.rpcUrls[0],
-
-  // Additional
-  [CHAINS.OKEX]: 'https://exchainrpc.okex.org',
-  [CHAINS.AVALANCHE]: 'https://api.avax.network/ext/bc/C/rpc',
-  [CHAINS.FSN]: 'https://fsnmainnet2.anyswap.exchange',
-  [CHAINS.KOV]: 'https://kovan.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-  [CHAINS.ONE]: 'https://api.harmony.one',
-  [CHAINS.ONET]: 'https://api.s0.b.hmny.io',
+  [ChainId.ROP]: process.env.REACT_APP_RPC_URL_ROPSTEN,
+  [ChainId.RIN]: process.env.REACT_APP_RPC_URL_RINKEBY,
+  [ChainId.GOR]: process.env.REACT_APP_RPC_URL_GORLI,
+  [ChainId.KOV]: process.env.REACT_APP_RPC_URL_KOVAN,
+  [ChainId.ARBT]: process.env.REACT_APP_RPC_URL_ARBITRUM_RINKEBY,
+  [ChainId.OPTT]: process.env.REACT_APP_RPC_URL_OPTIMISM_KOVAN,
+  [ChainId.MUM]: process.env.REACT_APP_RPC_URL_POLYGON_MUMBAI,
+  [ChainId.BSCT]: process.env.REACT_APP_RPC_URL_BSC_TESTNET,
 }
 
 // based on:
 // - https://github.com/sushiswap/sushiswap-sdk/blob/canary/src/constants/addresses.ts#L323
 // - https://github.com/joshstevens19/ethereum-multicall#multicall-contracts
 const MULTICALL_ADDRESSES: { [chainId: number]: string } = {
   // Mainnet
-  [CHAINS.MAINNET]: '0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696',
-  [CHAINS.POLYGON]: '0x02817C1e3543c2d908a590F5dB6bc97f933dB4BD',
-  [CHAINS.BSC]: '0xa9193376D09C7f31283C54e56D013fCF370Cd9D9',
-  [CHAINS.XDAI]: '0x67dA5f2FfaDDfF067AB9d5F025F8810634d84287',
-  [CHAINS.FANTOM]: '0x22D4cF72C45F8198CfbF4B568dBdB5A85e8DC0B5',
-  [CHAINS.ARBITRUM]: '0x80C7DD17B01855a6D2347444a0FCC36136a314de',
-  [CHAINS.OKEX]: '0xF4d73326C13a4Fc5FD7A064217e12780e9Bd62c3',
-  [CHAINS.AVALANCHE]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
-  [CHAINS.FSN]: '0x0769fd68dFb93167989C6f7254cd0D766Fb2841F',
-  [CHAINS.KOV]: '0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696',
-  [CHAINS.ONE]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
-  // [CHAINS.OPTIMISTIC]: '',
+  [ChainId.ETH]: '0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696',
+  [ChainId.POL]: '0x02817C1e3543c2d908a590F5dB6bc97f933dB4BD',
+  [ChainId.BSC]: '0xa9193376D09C7f31283C54e56D013fCF370Cd9D9',
+  [ChainId.DAI]: '0x67dA5f2FfaDDfF067AB9d5F025F8810634d84287',
+  [ChainId.FTM]: '0x22D4cF72C45F8198CfbF4B568dBdB5A85e8DC0B5',
+  [ChainId.ARB]: '0x80C7DD17B01855a6D2347444a0FCC36136a314de',
+  [ChainId.OKT]: '0xF4d73326C13a4Fc5FD7A064217e12780e9Bd62c3',
+  [ChainId.AVA]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
+  [ChainId.FSN]: '0x0769fd68dFb93167989C6f7254cd0D766Fb2841F',
+  [ChainId.KOV]: '0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696',
+  [ChainId.ONE]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
```

Comment:
I noticed this one (HARMONY ONE) failing a lot today while bugfixing. Is there an alternative?

Reply:
For the multicall contract or the RPC?

Reply:
Harmony is also only tested on staging and not in production, so I think we can skip this here.